### DISCUSSION TITLE
Elementos por defecto en componentes

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#1144] [Proposals]
### Goal

Evaluar cuál debería ser la estrategia a la hora de crear nuevos componentes en la librería con respecto a la visibilidad de sus elementos.

* **Estrategia 1**: Los componentes presentarán por defecto únicamente sus elementos mínimos, esto permite que cuando un componente esté desenlazado de la librería o una de sus propiedades se modifique no aparezcan elementos indeseados al hacer update.
* **Estrategia 2**: Mostrar los elementos que se espera / se conoce (en caso de un refactor) que sean los mas usados para evitar que el usuario tenga trabajo adicional tocando propiedades de la librería.

### Context

Recientemente ha ocurrido con componentes que estaban desenlazados de la librería que al volver a enlazarlos han aparecido elementos indeseados que vienen por defecto en el componente. Esto puede suponer un retrabajo a la hora de afrontar actualización de archivos legacy que usan mística o breaking changes de la librería.

### Impact

En función de la estrategia elegida:

* Mejoras en mantenimiento, evitar breaking changes
* Menor uso del panel de propiedades

### Discussion Questions

* Cual debería ser la estrategia idónea a seguir
* Existe una tercera opción para tener lo mejor de los dos mundos

**Comment by yceballost** [comment]
Vamos con la estrategia 1 siempre que sea ocultando actuales y no mostrar nada nuevo para evitar que aparezca cosas indeseadas al actualizar
